projects/BFDR.Tests/BirthRecord_Should.cs

```diff
@@ -252,25 +252,25 @@ public void Set_MotherEthnicity1()
 
       SetterBirthRecord.MotherEthnicity1Helper = "N";
       Dictionary<string, string> CodeN = new Dictionary<string, string>();
-      CodeN.Add("code", VR.ValueSets.HispanicNoUnknown.Codes[0, 0]);
-      CodeN.Add("display", VR.ValueSets.HispanicNoUnknown.Codes[0, 1]);
-      CodeN.Add("system", VR.ValueSets.HispanicNoUnknown.Codes[0, 2]);
+      CodeN.Add("code", VR.ValueSets.YesNoUnknown.Codes[0, 0]);
+      CodeN.Add("display", VR.ValueSets.YesNoUnknown.Codes[0, 1]);
+      CodeN.Add("system", VR.ValueSets.YesNoUnknown.Codes[0, 2]);
```

Comment:
I didn't see in the changes that ethnicity uses YesNoUnknown now. Can you send a link to the ballot version where it shows this change?

Reply:
[older ballot version](https://hl7.org/fhir/us/vr-common-library/2024Jan/StructureDefinition-input-race-and-ethnicity-vr.html#terminology-bindings-differential)

[current build](https://build.fhir.org/ig/HL7/vr-common-library/StructureDefinition-input-race-and-ethnicity-vr.html#terminology-bindings-differential)